odata-json-format/meta.yaml

```diff
@@ -6,3 +6,9 @@ previousStage: n/a
 pubdate: 14 July 2023
 pubdateISO: '2023-07-14'
 copyright: © OASIS Open 2023
+description:
+  "The Open Data Protocol (OData) for representing and interacting with structured
```

Comment:
Why do we need the double-quotes here?

Reply:
Otherwise the row
```
protocol is in OData Version 4.02 Part 1: Protocol.
```
is interpreted as YAML key-value pair.

Reply:
[Block Scalars](https://yaml-multiline.info/) seem to do the trick